package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "audion",
+  "version": "1.0.0",
+  "description": "Traces web audio API calls.",
```

Comment:
nit: A Chrome DevTools extension traces Web Audio API calls and visualizes in the DevTools.